add_car.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+require_once "vendor/autoload.php";
+require_once "src/utils.php";
+
+use CarStore\UpdateCars;
+use CarStore\Car;
+
+session_start();
+
+if (
+    isset($_POST["name"]) &&
+    isset($_POST["year_made"]) &&
+    isset($_POST["zero_sixty"]) &&
+    isset($_POST["price"]) &&
+    isset($_POST["brand"])
+) {
+    $name = $_POST["name"];
+    $year_made = $_POST["year_made"];
+    $zero_sixty = $_POST["zero_sixty"];
+    $price = $_POST["price"];
+    $brand = $_POST["brand"];
```

Comment:
You should do some basic validation at this point. 

A few things to check:
1) The lengths of the inputted data don't exceed what your database can store (based on the length option in the structure tab of sequel ace)
2) The fields aren't empty
3) That the year and price fields actually contain numbers

Display appropriate error messages based on the validation and only try to add the car if all of your requirements are met

Reply:
Consider making this validation into a function or object as well, it might be useful when it comes to making an edit form